Code/GraphMol/Depictor/EmbeddedFrag.cpp

```diff
@@ -350,10 +333,9 @@ void EmbeddedFrag::setupNewNeighs() {  // const RDKit::ROMol *mol) {
   PRECONDITION(dp_mol, "");
 
   RDKit::ROMol::ADJ_ITER nbrIdx, endNbrs;
-  INT_EATOM_MAP_I eci;
   d_attachPts.clear();
-  for (eci = d_eatoms.begin(); eci != d_eatoms.end(); eci++) {
-    unsigned int aid = eci->first;
+  for (const auto &eci : d_eatoms) {
+    unsigned int aid = eci.first;
```

Comment:
You've been consolidating to auto in these cases, might as well be consistent.

Reply:
yep. will do